front/app/api/projects/types.ts

```diff
@@ -137,7 +140,8 @@ export type CommentingDisabledReason =
   | 'project_inactive'
   | 'not_supported'
   | 'commenting_disabled'
-  | PermissionsDisabledReason;
+  | PermissionsDisabledReason
+  | null;
```

Comment:
You don't need to add `null` here- this is already covered by the type definition of `ActionDescriptor`, see `app/utils/actionDescriptors.ts`

Reply:
Clear! I missed this somehow. And also didn't get why isFixableByAuthentication was not complaining when we were passing disabled_reason that could be null before but then I also missed we do checks in places where it's used to ensure enabled is false (and so disabled_reason not null) by the time isFixableByAuthentication gets called.